hse/pom.xml

```diff
@@ -38,9 +38,9 @@ This code is derived from the ycsb project.
       <version>0.10.3</version>
     </dependency>
     <dependency>
-      <groupId>com.micron.hse_project</groupId>
+      <groupId>io.github.hse_project</groupId>
```

Comment:
Educate me.  Why are we making this change?

Reply:
Essentially the issue is that OSSRH, the repo we will publish to, requires you to verify domain ownership. One way to do this is through DNS records.

The chances we get a top level DNS record for micron.com? Probably less than 0.

Luckily OSSRH allows us to use our GitHub website for the package name.

This change happened in coordination with Steve and Bhavesh which is why you are just now finding out about it.